web_src/js/components/RepoContributors.vue

```diff
@@ -0,0 +1,353 @@
+<template>
+  <div>
+    <h2 class="ui header">
+      <relative-time
+        v-if="dateFrom !== null"
+        format="datetime"
+        year="numeric"
+        month="short"
+        day="numeric"
+        weekday=""
+        :datetime="dateFrom"
+      >
+        {{ dateFrom }}
+      </relative-time>
+      {{ isLoading ? "Loading contributions..." : errorText ? "Loading contributions failed :(": "-" }}
+      <relative-time
+        v-if="dateUntil !== null"
+        format="datetime"
+        year="numeric"
+        month="short"
+        day="numeric"
+        weekday=""
+        :datetime="dateUntil"
+      >
+        {{ dateUntil }}
+      </relative-time>
+
+      <div class="ui right">
+        <!-- Contribution type -->
+        <div class="ui dropdown jump" id="dropdown">
+          <div class="ui basic compact button">
+            <span class="text">
+              Contribution type: <strong>{{ type }}</strong>
+              <svg-icon name="octicon-triangle-down" :size="14"/>
+            </span>
+          </div>
+          <div class="menu">
+            <div :class="type === 'commits' ? 'active item' : 'item'">
+              Commits
+            </div>
+            <div :class="type === 'additions' ? 'active item' : 'item'">
+              Additions
+            </div>
+            <div :class="type === 'deletions' ? 'active item' : 'item'">
+              Deletions
+            </div>
+          </div>
+        </div>
+      </div>
+    </h2>
+    <div class="divider"/>
+    <div style="height: 380px" class="gt-df">
```

Comment:
No inline styles please. Give this a class name and use `<style scoped>` at the end of this file to style it.